services/application.js

```diff
@@ -149,7 +155,7 @@ class Application {
             $set: {reviewComment: document.comment, status: REJECTED, updatedAt: history.dateTime},
             $push: {history}
         });
-        return (updated.modifiedCount && updated.modifiedCount > 0) ? await this.dbService.find(APPLICATION, {_id: document._id}) : null;
+        return updated.modifiedCount && updated.modifiedCount > 0 ? this.getApplicationById(document._id) : null;
```

Comment:
Let's make this null safe as well, the same as the previous comment

Reply:
updated